SWLOR.Game.Server/Data/Migrations/2018-10-25.5.sql

```diff
@@ -0,0 +1,321 @@
+﻿INSERT INTO dbo.CustomEffects (CustomEffectID ,
+					Name ,
+					IconID ,
+					ScriptHandler ,
+					StartMessage ,
+					ContinueMessage ,
+					WornOffMessage ,
+					IsStance
+				)
+
+VALUES (25 , -- CustomEffectID 
+		N'Force Leech' ,	--	Name 
+		0 ,	--	IconID 
+		N'ForceLeechEffect' ,	--	ScriptHandler 
+		N'You feel your life leaving you' ,	--	StartMessage 
+		N'You continue to grow weaker' ,	--	ContinueMessage 
+		N'You are starting to feel normal again' ,	--	WornOffMessage 
+		0 	--	IsStance
```

Comment:
I made a change to this table that you don't seem to have. Instead of IsStance it's now CustomEffectCategoryID. You'll want to sync up with master and try to fix this. Sorry, forgot to mention that. Message me and we'll go through how to do that tomorrow.